reV/supply_curve/supply_curve.py

```diff
@@ -966,10 +981,34 @@ def _check_feature_capacity(self, avail_cap_frac=1):
             fc = TF.feature_capacity(self._trans_table, **kwargs)
             self._trans_table = self._trans_table.merge(fc, on='trans_gid')
 
+    def _adjust_sort_inputs(self, columns, sort_on, consider_friction):
+        """Add extra output columns and set `sort_on` value, if needed. """
+        # These are essentially should-be-defaults that are not
+        # backwards-compatible, so have to explicitly check for them
+        extra_cols = {'poi_lat', 'poi_lon', 'reinforcement_poi_lat',
+                      'reinforcement_poi_lon', 'eos_mult', 'reg_mult'}
+
+        if consider_friction and 'total_lcoe_friction' in self._trans_table:
+            extra_cols.add('total_lcoe_friction')
```

Comment:
why don't you just iterate through these extra cols? What about:
```
extra_cols = {all, the, options}
extra_cols = set(col for col in extra_cols if col in self._trans_table)

if 'reinforcement_cost' in self._trans_table:
    sort_on = sort_on or "lcoe_no_reinforcement"

Reply:
Yea that's a great point. I've changed the code to match that style and split apart the unrelated functionality of determining the output columns and the `sort_on` column into separate functions